simpleflow/activity.py

```diff
@@ -37,7 +37,7 @@ def wrap(func):
     return wrap
 
 
-class Activity(object):
+class ActivityType(object):
```

Comment:
Yay for that, as discussed yesterday ; there's a vocab problem here, if anybody has a better idea, speak now!